mkosi/config.py

```diff
@@ -44,13 +47,38 @@ def parse_boolean(s: str) -> bool:
     die(f"Invalid boolean literal: {s!r}")
 
 
+def parse_path(value: str, *, required: bool, absolute: bool = True, expanduser: bool = True, expandvars: bool = True) -> Path:
+    if expandvars:
+        path = Path(string.Template(value).substitute(os.environ))
```

Comment:
This can raise `ValueError` (if a $ is incorrectly used) or `KeyError` (if an environment variable is no set).

Should I encapsulate the exception? Use `.expandvars()` which would leave those intact ?

Reply:
Probably `expandvars()`